### PR TITLE
chore: add in issue templates and badges

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

**Comment by ckipp01** [comment]
I have some legit failures here that I'm not sure how they passed CI before.

**Review by tgodzik — APPROVED** [review]
LGTM! The failures don't seem to be related.

**Comment by ckipp01** [comment]
> Looks like tests started using the last release RC version: Failed to execute goal ch.epfl.scala:bloop-maven-plugin:2.0.0-RC3:bloopInstall

Yea... this was my silly mistake. I'm fixing it. I can't call it like this as it will pull the latest, not the self-published one. I'll have another pr soon that fixes this (because I also found another bug). So once I get that other one merged I'll then rebase this on it.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: 🐛 Bug Report
```
[the remark this answers is not in the document]
Comment (ckipp01) [reply .github/ISSUE_TEMPLATE/bug_report.yml:1]: Ah good idea, I'll add that.

**Pull request merged**